js/test/Ice/operations/Client.ts

```diff
@@ -38,6 +38,8 @@ export class Client extends TestHelper {
         try {
             const [properties] = this.createTestProperties(args);
             properties.setProperty("Ice.BatchAutoFlushSize", "100");
+            properties.setProperty("Ice.Trace.Protocol", "1");
```

Comment:
Protocol=1 and Network=3 sounds excessive.

Reply:
This is to ensure that we exercise this code paths.